### PR TITLE
fixing JSDOCS for Sorts Folder the folder that been modified are Alph…

### PR DESCRIPTION
**Pull request by mohmmadAyesh**
…aNumerical,BeadSort,BogoSort,BubbleSort,CocktailShakerSort,CountinfSort,FindSecondsLargestElementSort,FisherYatesShuffle

[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### Describe your change:

- [ ] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ x] Documentation change?

### Checklist:

- [ x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [x ] This pull request is all my own work -- I have not plagiarized.
- [x ] I know that pull requests will not be merged if they fail the automated tests.
- [ ] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x ] All new JavaScript files are placed inside an existing directory.
- [ x] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [x ] All new algorithms have a URL in their comments that points to Wikipedia or another similar explanation.
- [x ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.
Hi I added JSDOCS to Files that missing it in Sorts Folder this is a part of solving issue #1342 `Fixes:#{1342}` Hope you accept it

### CODE REVIEW

File: Sorts/BeadSort.js [hunk 1]
```diff
@@ -7,6 +7,9 @@
  * NOTE: It only works for arrays of positive integers.
  *
  * Wikipedia: https://en.wikipedia.org/wiki/Bead_sort
+ * @param {number[]} sequence An array of positive integers to be sorted using Bead Sort.
```
Comment (appgurueu) [inline Sorts/BeadSort.js:10]: The "using Bead Sort" is not helpful here or in the next line.

File: Sorts/BogoSort.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
 /**
  * Checks whether the given array is sorted in ascending order.
+ * @param {number[]} array The array to be checked for sorted order.
```
Comment (appgurueu) [inline Sorts/BogoSort.js:3]: `number[]` is wrong. Any comparable objects are fine. (Strings would be fine as well, for example, though the order may not be what you expect.)

The comments are also a bit redundant. You're effectively stating the same thing three times.

File: Sorts/BogoSort.js [hunk 3]
```diff
@@ -13,6 +15,7 @@ export function isSorted(array) {
 
 /**
  * Shuffles the given array randomly in place.
+ * @param {any[]} array The array to be shuffled.
```
Comment (appgurueu) [inline Sorts/BogoSort.js:18]: How does this comment help?

File: Sorts/BubbleSort.js [hunk 4]
```diff
@@ -42,6 +44,8 @@ export function bubbleSort(items) {
 
 /**
  * Using a while loop and a for loop.
+ * @param {number[]} arr The array to be sorted.
```
Comment (appgurueu) [inline Sorts/BubbleSort.js:47]: Note that the array is sorted *in-place*.

File: Sorts/CocktailShakerSort.js [hunk 5]
```diff
@@ -7,6 +7,9 @@
  *
  * Wikipedia (Cocktail Shaker Sort): https://en.wikipedia.org/wiki/Cocktail_shaker_sort
  * Wikipedia (Bubble Sort): https://en.wikipedia.org/wiki/Bubble_sort
+ *
+ * @param {number[]} items The array of numbers to be sorted.
```
Comment (appgurueu) [inline Sorts/CocktailShakerSort.js:11]: Again, please note that the sorting is *in-place*

File: Sorts/FisherYatesShuffle.js [hunk 6]
```diff
@@ -1,3 +1,8 @@
+/**
+ * Shuffles the elements of the given array randomly in place.
```
Comment (appgurueu) [inline Sorts/FisherYatesShuffle.js:2]: Again you only really need one of the comments.

**Comment by mohmmadAyesh** [comment]
This branch merged with fix-docs

**Pull request closed without merging**